### PR TITLE
Resizes interactive to keep default aspect ratio when height or width of browser window is changed.

### PR DESCRIPTION
**Pull request by eireland**
Demo [here](https://activity-player.concord.org/branch/default-aspect-ratio/?activity=interactive-sizing-demo-question-interactive&page=page_124109)

### CODE REVIEW

File: src/components/activity-page/managed-interactive/managed-interactive.tsx [hunk 1]
```diff
@@ -153,7 +153,8 @@ export const ManagedInteractive: React.ForwardRefExoticComponent<IProps> = forwa
 
   const divTarget = React.useRef(null);
   const divSize: any = useSize(divTarget);
-  const containerWidth: number = divSize?.width;
+  let containerWidth: number | string;
+  containerWidth = "100%";
```
Comment (kswenson) [inline src/components/activity-page/managed-interactive/managed-interactive.tsx:157]:
```suggestion
  let containerWidth: number | string = "100%";
```

File: src/components/activity-page/managed-interactive/managed-interactive.tsx [hunk 2]
```diff
@@ -162,8 +163,12 @@ export const ManagedInteractive: React.ForwardRefExoticComponent<IProps> = forwa
       proposedHeight = divSize?.width / aspectRatio;
       break;
     default:
-      proposedHeight = divSize?.width / aspectRatio;
-      break;
+      if (divSize?.width / aspectRatio > screenHeight.dynamicHeight) {
+        proposedHeight = screenHeight.dynamicHeight - 15;
```
Comment (kswenson) [inline src/components/activity-page/managed-interactive/managed-interactive.tsx:167]: Named constant and/or comment to explain magic number "15" would  be useful.

File: src/components/activity-page/managed-interactive/managed-interactive.tsx [hunk 3]
```diff
@@ -162,8 +163,12 @@ export const ManagedInteractive: React.ForwardRefExoticComponent<IProps> = forwa
...
+        containerWidth = (proposedHeight * aspectRatio);
+      } else {
+        proposedHeight = divSize?.width / kDefaultAspectRatio;
+      }      break;
```
Comment (kswenson) [inline src/components/activity-page/managed-interactive/managed-interactive.tsx:171]:
```suggestion
      }
      break;
```

**Review by emcelroy — COMMENTED** [review]
Demo works well. I agree with Kirk's suggestions.

**Review by kswenson — APPROVED** [review]
👍 Changes look good

**Pull request merged**